manifest/signature.go

```diff
@@ -74,10 +89,16 @@ func (m *ModuleHashes) HashModule(modules *pbsubstreams.Modules, module *pbsubst
 	buf.WriteString(modules.Binaries[module.BinaryIndex].Type)
 	buf.Write(modules.Binaries[module.BinaryIndex].Content)
 
+	param, ok := m.params[module.Name]
+	if ok {
+		buf.WriteString("params")
+		buf.WriteString(param) //TODO: parse and sort?
+	}
+
 	buf.WriteString("inputs")
 	for _, input := range module.Inputs {
-		buf.WriteString(inputName(input))
-		buf.WriteString(inputValue(input))
+		buf.WriteString(m.inputName(input, module))
+		buf.WriteString(m.inputValue(input, module))
 	}
```

Comment:
In this file, the only necessary thing is the addition of the two `case` clauses to add support for `*pbsubstreams.Module_Input_Params_`. The loop through the inputs will cover the params, as well as the value for the params. No need for an additional "params" section, no need for a member `params`.
The params will always be fit _directly_ in the module's `Module_Input_Params_`, and analyzed there.
Did I miss something?